src/renderer/views/pools/Pools.types.ts

```diff
@@ -22,6 +24,9 @@ export type PricePool = {
   poolData: PoolData
 }
 
+export type PricePool$ = Rx.Observable<PricePool>
+export type PricePoolRD = RD.RemoteData<Error, PricePool>
```

Comment:
hm. this time i could not fins usage of `PricePoolRD` 😅

Reply:
Yeah, that happens if you tried an issue, but using different way before :sweat_smile: